extensions/0832/SuperAlert.js

```diff
@@ -0,0 +1,317 @@
+(function (Scratch) {
+    'use strict';
+    var b = false;
+    var c;
+    Scratch.translate.setup({
+        zh: {
+            name: '弹窗',
+            show_alert: '显示弹窗',
+            hide_alert: '隐藏弹窗',
+            set_title: '设置弹窗标题为 [L]',
+            set_content: '设置弹窗内容为 [M]',
+            set_position: '设置弹窗位置为 X: [X] Y: [Y]',
+            set_text: '设置 [T] 文本为 [C]',
+            align_title: '将标题文本对齐到 [A]',
+            align_content: '将内容文本对齐到 [A]',
+            set_font: '设置 [T] 字体为 [F]',
+            set_text_color: '设置 [T] 文本颜色为 [C]',
+            set_text_size: '将 [T] 文本大小设置为 [S]px'
+        },
+    });
+
+    function d() {
+        c = document.createElement("div");
+        c.style.position = "fixed";
+        c.style.backgroundColor = "white";
+        c.style.border = "1px solid black";
+        c.style.top = "50%";
+        c.style.left = "50%";
+        c.style.transform = "translate(-50%, -50%)";
+        c.style.display = "none";
+        c.style.borderRadius = "10px";
+        c.style.color = "white";
+        c.style.backgroundColor = "black";
+        c.style.border = "1px solid white";
+        c.style.padding = "20px";
+        c.style.textAlign = "center";
+        c.style.transition = "opacity 0.3s, transform 0.3s";
+        c.style.opacity = "0";
+        document.body.appendChild(c);
+        setTimeout(function () {
+            b = true;
+            c.style.opacity = "1";
+            c.style.transform = "translate(-50%, -50%) scale(1)";
+            c.style.display = "block";
+        }, 100);
+    }
+
+    function e() {
+        c.style.opacity = "0";
+        c.style.transform = "translate(-50%, -50%) scale(0)";
+        setTimeout(function () {
+            document.body.removeChild(c);
+            b = false;
+        }, 300);
+    }
+
+    class f {
+        constructor() {
+            this.g = "";
+            this.h = "";
+            this.xPos = -50; // Default X position
+            this.yPos = -50; // Default Y position
+            this.titleFont = "Arial";
+            this.contentFont = "Arial";
+            this.titleColor = "white";
+            this.contentColor = "white";
+            this.titleSize = 24;
+            this.contentSize = 16;
+            this.outlineSize = 1;
+        }
+        d() {
```

Comment:
It appears there's no way to open a text box with it's contents already defined; it'll always be empty for a few frames before content is added. Is there a way around this?